bin/genesis

```diff
@@ -528,7 +534,7 @@ EOF
 
 sub kit_yaml_files_in {
 	my ($env, $dir, @subkits) = @_;
-	$env =~ s/\.ya?ml$//;
+	$env =~ s/\.yml$//;
```

Comment:
Why no love for `.yaml`?

Reply:
https://github.com/starkandwayne/genesis/blob/master/bin/genesis#L629-L656

Reply:
Annoying.